default/AI/DiplomaticCorp.py

```diff
@@ -0,0 +1,78 @@
+"""Handle diplomatic messages and response determination."""
+
+import random
+
+import freeOrionAIInterface as fo  # pylint: disable=import-error
+import FreeOrionAI as foAI
+import AIstate
+from freeorion_tools import UserString, UserStringList, chat_on_error, print_error
+
+
+@chat_on_error
+def handle_diplomatic_message(message):  # pylint: disable=invalid-name
```

Comment:
ah, probably not, when I moved the function over from FreeOrionAI I changed the name but forgot to remove the comment.